docs/source/specs/openapi.json

```diff
@@ -525,6 +525,74 @@
           }
         }
       },
+      "get": {
+        "tags": [
+          "Group"
+        ],
+        "summary": "Get a list of principals from a group in the tenant",
+        "operationId": "getPrincipalsFromGroup",
+        "parameters": [
+          {
+            "name": "uuid",
+            "in": "path",
+            "description": "ID of group from which to get principals",
+            "required": true,
+            "schema": {
+              "type": "string",
+              "format": "uuid"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List of principals attached to group",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
```

Comment:
I think here for the 200 we may just need:
```
          "200": {
            "description": "List of principals attached to group",
            "content": {
              "application/json": {
                "schema": {
                  "$ref": "#/components/schemas/PrincipalPagination"
                }
              }
            }
          }
```
The output in https://editor.swagger.io/ for this as is doesn't match the actual response, as it nests everything in a principal lists.